khmer/khmer_args.py

```diff
@@ -55,9 +70,9 @@ def optimal_size(num_kmers, mem_cap=None, fp_rate=None):
       - fp_rate: the desired false positive rate [optional, conflicts with M]
     """
     if all((num_kmers is not None, mem_cap is not None, fp_rate is None)):
-        return estimate_optimal_with_K_and_M(num_kmers, mem_cap)
+        return estimate_optimal_with_k_and_m(num_kmers, mem_cap)
```

Comment:
why lowercase 'M'? I am -0 on this, for two reasons --

- `-M` is the command line option we're using!
- mathematically, `M` and `N` are things we (and others) use in equations, and they are typically uppercase.
- well, and I see no point to this change.